tests/unit_tests/CMakeLists.txt

```diff
@@ -8,7 +8,10 @@
 add_subdirectory( covfie )
 add_subdirectory( cpu )
 add_subdirectory( io )
-add_subdirectory( svgtools )
+
+if ( DETRAY_SVG_DISPLAY )
```

Comment:
We only reach this point if ```DETRAY_BUILD_TESTING``` is ```ON```, right? So is this not just always true?
Actually, the inclusion of svgtools should probably have been in ```tests/CMakeLists.txt```, but that is another issue...